MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,853 @@
+---
+layout: post
+title: Change the data editor settings in .NET MAUI DataForm control | Syncfusion
+description: Learn about to change the Syncfusion .NET MAUI DataForm (SfDataForm) settings in mobile and desktop applications from a single shared codebase.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Change the data editor settings (SfDataForm)
+
+## Auto generate data editor 
+
+By default, the `SfDataForm.Items` will be generated based on the property type. For example, the `DataFormTextItem` will be generated based on the property type in `DataObject`. 
+The `DataFormItem` generation depends on the type and attribute defined for the property.
+You can customize the auto generated editor, label and group settings by using the `GenerateDataFormItem` event.
```

Comment:
You can customize the auto-generated editor, label, and group settings by using the `GenerateDataFormItem` event.

Reply:
updated